homeassistant/components/devolo_home_network/switch.py

```diff
@@ -117,7 +117,9 @@ async def async_turn_on(self, **kwargs: Any) -> None:
         except DevicePasswordProtected as ex:
             self.entry.async_start_reauth(self.hass)
             raise HomeAssistantError(
-                f"Device {self.entry.title} require re-authenticatication to set or change the password"
+                translation_domain=DOMAIN,
+                translation_key="password_protected",
+                translation_placeholders={"title": self.entry.title},
```

Comment:
I think we should still pass a non-translated string:
```suggestion
                f"Device {self.entry.title} require re-authenticatication to set or change the password",
                translation_domain=DOMAIN,
                translation_key="password_protected",
                translation_placeholders={"title": self.entry.title},
```

Reply:
Interesting, that I passed it everywhere else but removed it here...